src/main/java/com/minecolonies/structures/event/RenderEventHandler.java

```diff
@@ -1,8 +1,10 @@
 package com.minecolonies.structures.event;
 
 import com.minecolonies.api.util.BlockUtils;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.util.BlockUtils'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)